lib/ash/filter/filter.ex

```diff
@@ -732,6 +732,13 @@ defmodule Ash.Filter do
     end
   end
 
+  defp add_expression_part({:is_nil, field}, context, expression) do
```

Comment:
Right now you'd do it with `filter(field: [is_nil: true])`. `filer(is_nil: :field)` looks really nice though, good addition.

Reply:
Good to know, added support for the `is_nil: true` version.